src/main/java/com/code_intelligence/jazzer/mutation/mutator/collection/ChunkMutations.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2023 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.code_intelligence.jazzer.mutation.mutator.collection;
+
+import com.code_intelligence.jazzer.mutation.api.PseudoRandom;
+import com.code_intelligence.jazzer.mutation.api.SerializingMutator;
+import com.code_intelligence.jazzer.mutation.support.Preconditions;
+import java.util.AbstractList;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+// Based on (Apache-2.0)
+// https://github.com/google/fuzztest/blob/f81257ed70ec7b9c191b633588cb6e39c42da5e4/fuzztest/internal/domains/container_mutation_helpers.h
+final class ChunkMutations {
+  private ChunkMutations() {}
+
+  static <T> void deleteRandomChunk(List<T> list, int minSize, PseudoRandom prng) {
+    Preconditions.require(list.size() > minSize);
+
+    int minFinalSize = Math.max(minSize, list.size() / 2);
+    int chunkSize = prng.closedRangeBiasedTowardsSmall(1, list.size() - minFinalSize);
+    int chunkOffset = prng.closedRange(0, list.size() - chunkSize);
+
+    list.subList(chunkOffset, chunkOffset + chunkSize).clear();
+  }
+
+  static <T> void insertRandomChunk(
+      List<T> list, int maxSize, SerializingMutator<T> elementMutator, PseudoRandom prng) {
+    Preconditions.require(list.size() < maxSize);
+
+    int chunkSize = prng.closedRangeBiasedTowardsSmall(1, maxSize - list.size());
+    int chunkOffset = prng.closedRange(0, list.size());
```

Comment:
nit: calling list.size() 3 times in a row

Reply:
Fixed.